src/api/v1/api.json

```diff
@@ -325,8 +325,7 @@
     "POST": {
       "isImplemented": true,
       "permissions": {
-        "VOLUNTEER": "volunteer_logs-own:write",
-        "VOLUNTEER_ADMIN": ["volunteer_logs-sibling:write", "volunteer_logs-own:write"],
+        "VOLUNTEER_ADMIN": ["volunteer_logs-sibling:write"],
```

Comment:
This needs to retain the `own` scopes for both `VOLUNTEER_ADMIN` and `VOLUNTEER` users.